gordo_components/cli/client.py

```diff
@@ -28,7 +28,7 @@
 @click.option("--port", help="Port the server is running on", default=443)
 @click.option("--scheme", help="tcp/http/https", default="https")
 @click.option("--gordo-version", help="Version of gordo", default="v0")
-@click.option("--batch-size", help="How many samples to send", default=100000)
```

Comment:
Is this what we _really_ want?! :fearful:

Reply:
Whoops... that was just for my WIP testing, things are looking good now so I'll make that get outa here.